packages/lodestar/src/util/forkDigestContext.ts

```diff
@@ -0,0 +1,41 @@
+import {computeForkDigest} from "@chainsafe/lodestar-beacon-state-transition";
+import {ForkName, IBeaconConfig} from "@chainsafe/lodestar-config";
+import {ForkDigest, Root} from "@chainsafe/lodestar-types";
+import {mapValues} from "@chainsafe/lodestar-utils";
+import {toHexString} from "@chainsafe/ssz";
+
+type ForkDigestHex = string;
+
+export type IForkDigestContext = {
+  forkDigest2ForkName(forkDigest: ForkDigest | ForkDigestHex): ForkName | undefined;
+  forkName2ForkDigest(forkName: ForkName): ForkDigest;
```

Comment:
We can be certain that there will always be a ForkDigest available for known forks. Types reflect this.